packages/perseus/src/components/__stories__/input-with-examples.stories.tsx

```diff
@@ -1,53 +1,48 @@
-import * as React from "react";
+import {actions} from "@storybook/addon-actions";
```

Comment:
I think you want `action` rather than `actions`.

```suggestion
import {action} from "@storybook/addon-actions";
```

https://storybook.js.org/docs/essentials/actions#exports

Reply:
Fixed across the board. :)